### ISSUE TITLE
Feature: Talkgroup Hold

### ISSUE THREAD

**Issue by DSheirer** [DSheirer/sdrtrunk#50]
Add ability to hold on a specific talkgroup.  

When monitoring a system and a specific talkgroup is active, add ability to hold (only listen to) that specific talkgroup, until the hold is removed.

**Comment by DSheirer** [comment]
This feature is still open and hasn't been implemented yet.

**Comment by malchiuri** [comment]
Hello,
Your software is beautiful. Working fine all the time.
Talkgroup HOLD is important to remain on specific talkgroup on sudden event.
Is an "HOLD" button can make a temporary list with that specific talkgroup number, have Audio Priority = 1, and at the same time create blacklist with same list name, with Audio Priority = Do Not Monitor, to override and silence the other lists, until you release and let it run normal again.
It will help simplify monitoring without creating, arranging and folowing too many lists .
Thank you.

**Comment by FrostTheFox** [comment]
Just want to drop by with a +1 on this issue. I've converted at least a couple different people to SDRTrunk and while I personally host a calls node and don't have a need for this, the people I've converted are more used to traditional scanners. SDRTrunk's alias system is very powerful but it would be great to have this as an option for people who just want to casually hop between listening to a single talkgroup.

**Comment by PaulWeiss** [comment]
SDRTrunk is amazing software! Works as well, sometimes better than my Uniden SDS100 and Whistler TRX-1. SDRTrunk is easier to configure too!

+1 to add a hold talkgroup feature. This would make this software 100% complete in my view.

**Comment by Hermite123456** [comment]
> SDRTrunk is amazing software! Works as well, sometimes better than my Uniden SDS100 and Whistler TRX-1. SDRTrunk is easier to configure too!
> 
> +1 to add a hold talkgroup feature. This would make this software 100% complete in my view.

Totally agree. This would make the app pretty complete :)

**Comment by labomb** [comment]
Just wanted to add a +1 for a hold on talkgroup feature. The lack of this capability is the primary reason I still use op25 and dsdplus. Really enjoy sdrtrunk otherwise.

**Comment by edmk5000** [comment]
I am NOT a programmer but used Claude AI to write code that lets you HOLD on a talkgroup in Now Playing with P25 systems.  It also lets you save that held TG to the active alias list. I have never written or contributed to a project but would like to contribute this in hopes that this feature is useful. Claude AI also wrote a detailed description of what the feature does and how it works.   -> I am unsure of opinions, policy, or usability of using AI to write code to add a feature as in this case. It seems to be working well but surely needs to be tested and tweaked to fit the existing project. It is well-documented but understand using AI for this may be controversial.

I attached the text file the AI wrote describing what the feature is, how it works, what it wrote, and what files it made or modified.

In brief:
-The P25 Talkgroup Hold feature works by either clicking the talkgroup in traffic channel display or where it displays in the left Audio Channel. You can release the hold by clicking it again. The talkgroup shows as bold in the traffic channel and adds an (H) to the tg number in the audio channel.

-If you right-click the held talkgroup label next to the audio channel, you can store the held talkgroup to the active alias list.

In short, I worked with the Claude AI to implement this requested feature. I am only a user, not a programmer and did not program it myself. I would like to contribute the feature if it is useful but naive in submitting it as a pull request.

[TalkgroupHold_Summary.txt](https://github.com/user-attachments/files/27891181/TalkgroupHold_Summary.txt)

Ed
